_episodes_rmd/05-data-structures-part2.Rmd

```diff
@@ -5,13 +5,20 @@ exercises: 10
 questions:
 - "How can I manipulate a dataframe?"
 objectives:
-- "To learn how to manipulate a data.frame in memory"
-- "To tour some best practices of exploring and understanding a data frame when it is first loaded."
+- "Be able to add and remove rows and columns."
+- "Be able to remove rows with `na` values."
+- "Be able to append two data.frame"
```

Comment:
The shift to "data frame" also helps here, because then you can just say "data frames".